build/ansible/roles/pmm-images/tasks/main.yml

```diff
@@ -68,11 +80,10 @@
       group:
         name: "{{ item.name }}"
         gid: "{{ item.gid }}"
-        non_unique: true
       loop:
         - { name: pmm, gid: 1000 }
-        - { name: nginx, gid: 999 }
-        - { name: clickhouse, gid: 997 }
+        - { name: nginx, gid: 1002 }
+        - { name: clickhouse, gid: 1001 }
```

Comment:
These used to be created by the respective package installs. We create them only because we don't want to see uid/gid under 1000. They get removed in post-build.yml.